rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidConstants.java

```diff
@@ -126,4 +126,8 @@
     interface ScannerType {
         String SCANNER  = "scanner";
     }
+
+    interface SyncPreference {
+        String LAST_SYNC_DATE = "last_sync_date";
```

Comment:
Rename this https://github.com/onaio/rdt-standard/pull/825/files#r588313085.

Reply:
done